webapp/app.py

```diff
@@ -32,14 +34,16 @@
 )
 main_docs.init_app(app)
 
+init_tutorials(app, "/tutorials")
+
 
 @app.context_processor
 def utility_processor():
     return {"image": image_template}
 
 
 @app.route("/")
-def index():
+def index_temp():
```

Comment:
Can you name this something easier to understand?  What does `temp` mean?

Reply:
This was a change I did while debugging, didn't mean to leave it in there. Its updated now